### ISSUE TITLE
MISP Analyzer Tag and Sightings pull

### ISSUE THREAD

**Issue by robertnixon2003** [TheHive-Project/Cortex-Analyzers#175]
Is there any thoughts on adding to the MISP analyzer to pull all tags and number of sightings from the MISP events that match?

**Comment by 3c7** [comment]
Tags should already get displayed on matching events. Last time I checked the MISP api (dev workshop last November) it was not possible to grab the sightings - but I will check again. Sightings would be an enhancement indeed.

**Comment by robertnixon2003** [comment]
No tags are coming through on my install. Also when you export to MISP (share), shouldn't the sightings in TheHive be sent with the observables to MISP?

MISP Event:
![image](https://user-images.githubusercontent.com/20773170/35584066-d2fc4486-05c1-11e8-9755-add40214e308.png)

Cortex Report inside TheHive
![image](https://user-images.githubusercontent.com/20773170/35584119-062702a6-05c2-11e8-8b1b-1744c611e22e.png)

```
{
  "artifacts": [
    {
      "data": "http://xxx.xxx.xxx.xxx",
      "attributes": {
        "dataType": "url"
      }
    }
  ],
  "full": {
    "results": [
      {
        "url": "http://xxx.xxx.xxx.xxx",
        "name": "Unnamed",
        "result": [
          {
            "orgc_id": "2",
            "id": "4",
            "threat_level_id": "3",
            "uuid": "xxx",
            "Object": [],
            "Orgc": {
              "uuid": "xxx",
              "id": "2",
              "name": "PhishMe Intelligence Feed"
            },
            "RelatedEvent": [
              {
                "info": "Inquiry - CVE-2017-11882, Loki Bot",
                "id": "1"
              },
              {
                "info": "firehol_level1 feed",
                "id": "827"
              },
              {
                "info": " - Buying Order Quote - FormGrabber",
                "id": "828"
              },
              {
                "info": " - Inquiry Order - FormGrabber",
                "id": "5"
              },
              {
                "info": " - Payment Credited - FormGrabber",
                "id": "6"
              },
              {
                "info": " - See Attached Purchase Order - FormGrabber",
                "id": "683"
              },
              {
                "info": " - Provisional BC - FormGrabber",
                "id": "205"
              },
              {
                "info": " - Order - FormGrabber",
                "id": "671"
              },
              {
                "info": " - Payment - FormGrabber",
                "id": "131"
              },
              {
                "info": " - FedEx Office Delivery - FormGrabber",
                "id": "782"
              }
            ],
            "timestamp": "1517256532",
            "date": "2018-01-29",
            "info": " - New Inquiry - CVE-2017-11882, FormGrabber",
            "org_id": "1",
            "Galaxy": []
          },
          {
            "orgc_id": "2",
            "id": "1",
            "threat_level_id": "3",
            "uuid": "xxx",
            "Object": [],
            "Orgc": {
              "uuid": "x",
              "id": "2",
              "name": "xxx Feed"
            },
            "RelatedEvent": [
              {
                "info": " - New Inquiry - CVE-2017-11882, FormGrabber",
                "id": "4"
              }
            ],
            "timestamp": "1517326976",
            "date": "2018-01-30",
            "info": " - Inquiry - CVE-2017-11882, Loki Bot",
            "org_id": "1",
            "Galaxy": []
          }
        ]
      }
    ]
  },
  "summary": {
    "taxonomies": [
      {
        "predicate": "Search",
        "namespace": "MISP",
        "value": "\"2 event(s)\"",
        "level": "info"
      }
    ]
  },
  "success": true
}
```

**Comment by 3c7** [comment]
This is how it should look like:
<img width="209" alt="firefox_2018-01-31_15-15-33" src="https://user-images.githubusercontent.com/2846609/35627605-aefa55d6-0699-11e8-8665-d7da413e8ecf.png">

Are your tags exportable? Seems like they don't show up using the MISP api either.

**Comment by robertnixon2003** [comment]
You were right. I had to set this to false: MISP.incoming_tags_disabled_by_default
For those events since they came from another org.

What do you think about the sightings? If it is marked as sighted in TheHive should that carry over to MISP?

**Comment by saadkadhi** [comment]
Hi @robertnixon2003. Re 'What do you think about the sightings? If it is marked as sighted in TheHive should that carry over to MISP?', this has been planned for quite some time. It should be available with TheHive 3.1. We track it under [#366](https://github.com/TheHive-Project/TheHive/issues/366) in TheHive's repository.

If you have no other question/suggestion, please close this issue.
